owncloudData/src/test/java/com/owncloud/android/data/files/datasources/implementation/OCLocalFileDataSourceTest.kt

```diff
@@ -428,7 +436,8 @@ class OCLocalFileDataSourceTest {
     }
 
     @Test
-    fun `getFilesWithSyncInfoAvailableOfflineFromAccountAsFlow returns a Flow with an empty list when DAO returns a Flow with an empty list`() = runTest {
+    fun `getFilesWithSyncInfoAvailableOfflineFromAccountAsFlow returns a Flow with an empty list when DAO returns a Flow with an empty list`()
+    = runTest {
```

Comment:
Bitrise doesn't like this, so `=` can be moved at the end of the previous line

Reply:
Done